src/com/layout/col/col.js

```diff
@@ -0,0 +1,30 @@
+import {h, Component} from 'preact/preact';
+
+export default class Col extends Component {
+  constructor(props) {
+    super(props);
+  }
+
+  render({children, flex = 1, flexGrow, flexShrink, flexBasis = 100}) {
+    // let props = Object.assign({}, this.props, {
+    //   flex: undefined,
+    //   flexBasis: undefined,
+    //   flexGrow: undefined,
+    //   flexShrink: undefined
+    // });
```

Comment:
Values should never be initialized to `undefined`. Prefer `null`, as when we use `undefined`, you can't tell the difference between actual `undefined` values and user cleared values when debugging.

Reply:
I believe there was a reason for using `undefined` over `null`. The purpose of this code was to avoid showing some useless preact props in the DOM itself and setting it to `null` would just show it as a `null` value rather than hiding the value.

Either way I decided that it didn't really matter and so I commented it out, when really I should've just removed the code itself.

Reply:
Fair enough. It gave me a good opportunity to mention `null` is preferred.

Reply:
While I think `null` is preferable, here I think undefined is correct since the syntax asks for certain fields on `props` and they will be undefined not by setting that value but because they were actually undefined.